src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/InvalidDataContract.cs

```diff
@@ -34,12 +34,12 @@ public override void PrepareToRead(XmlReaderDelegator xmlReader)
             throw CreateInvalidDataContractException();
         }
 
-        public override object ReadXmlValue(XmlReaderDelegator xmlReader, XmlObjectSerializerReadContext context)
+        public override object ReadXmlValue(XmlReaderDelegator xmlReader, XmlObjectSerializerReadContext? context)
```

Comment:
how is ErrorMessage not null on the parameterless constructor?

Reply:
It doesn't look like this file is compiled. I can't find it in the .csproj. I must have changed it as part of a mass "find and replace" for `override object ReadXmlValue`. Will revert these changes.